### PR TITLE
Channel category updates

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/mattermost-webapp/pull/8839
- Updated Sidebar organization product docs to include details on moving channels to categories directly from the Channel Header

### CODE REVIEW

File: source/messaging/channel-settings.rst [hunk 1]
```diff
@@ -18,9 +18,8 @@ Channel Settings
   :target: https://mattermost.com/deploy
   :alt: Available for Mattermost Self-Hosted deployments.
 
-Notification preferences, channel header, channel purpose, and channel
-name are customizable for each channel. To access these settings, click
-the channel name at the top of the page to open the channel menu.
+Notification preferences, channel header, channel purpose, and channel name are customizable for each channel. To access these settings, select
```
Comment (justinegeffen) [inline source/messaging/channel-settings.rst]:
```suggestion
You can customize notification preferences, channel header, channel purpose, and channel name for each channel you're in. To access these settings, select
```

File: source/messaging/organizing-your-sidebar.rst [hunk 2]
```diff
@@ -107,7 +107,7 @@ All channels and Direct Message conversations in the deleted category move back
 Organizing channels in categories
 ---------------------------------
 
-Once you've created categories, you can move channels within them to organize your sidebar.
+Once you've created categories, you can move channels within them to organize your sidebar.
```
Comment (justinegeffen) [inline source/messaging/organizing-your-sidebar.rst:110]:
```suggestion
Once you've created categories, you can move channels around to organize your sidebar.
```

File: source/messaging/organizing-your-sidebar.rst [hunk 3]
```diff
@@ -129,9 +129,14 @@ Using the Mattermost Web or Desktop App, drag selected channels and/or Direct Me
 Moving selections
 ~~~~~~~~~~~~~~~~~
 
-In addition to selecting and dragging, you can also specify a destination for selected channels and/or Direct Messages using the **Move to** option under the **Channel options** icon in the sidebar.  
+In addition to selecting and dragging, you can specify a category destination for selected channels and/or Direct Messages using the **Move to** option under the **Channel options** icon in the sidebar.
```
Comment (justinegeffen) [inline source/messaging/organizing-your-sidebar.rst:132]:
```suggestion
In addition to selecting and dragging, you can specify a category destination for selected channels and/or Direct Messages. To do this, select the **Channel options** icon in the sidebar and then select **Move to**.
```

File: source/messaging/organizing-your-sidebar.rst [hunk 4]
```diff
@@ -129,9 +129,14 @@ Using the Mattermost Web or Desktop App, drag selected channels and/or Direct Me
 Moving selections
 ~~~~~~~~~~~~~~~~~
 
-In addition to selecting and dragging, you can also specify a destination for selected channels and/or Direct Messages using the **Move to** option under the **Channel options** icon in the sidebar.  
+In addition to selecting and dragging, you can specify a category destination for selected channels and/or Direct Messages using the **Move to** option under the **Channel options** icon in the sidebar.  
 
-    .. image:: ../images/multi-select-move.gif
+.. image:: ../images/multi-select-move.gif
+
+From Mattermost v6.1, you can also specify a category destination for the current channel or conversation using the **Move to** option directly from the Channel Header. Channels moved a category display a checkmark next to the category name.
```
Comment (justinegeffen) [inline source/messaging/organizing-your-sidebar.rst:136]:
```suggestion
From Mattermost v6.1, you can also specify a category destination for the current channel or conversation using the **Move to** option directly from the channel header. Channels that have been moved a category will display a checkmark next to the category name.
```

**Review by justinegeffen — COMMENTED** [review]
Did a little bit of re-wording for more natural language. Let me know what you think? It's only been applied to the changes so I may need to review the whole document before this is merged.

**Review by wiersgallak — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**